src/input.service.ts

```diff
@@ -343,8 +338,8 @@ export class InputService {
       // We want the gamepad to provide gamepad VK keyboard events rather than moving a
       // mouse like cursor. The gamepad will provide such keyboard events and provide
       // input to the DOM
-      (<any>navigator).gamepadInputEmulation = 'keyboard';
```

Comment:
this shud throw a typing error since `gamepadInputEmulation` is added by WinRT

Reply:
It's apparently in the new WinRT typings.